### DISCUSSION TITLE
关于背景颜色断层

### DISCUSSION THREAD

**Discussion by vercoo666** [amll-dev/applemusic-like-lyrics#225] [General]
背景颜色断层不知能否优化一下（还是跟显示设备有关？）

**Comment by Steve-xmh** [comment]
可能是渲染精度的问题，可以的话，使用你的浏览器（Chromuim系均可），访问 [chrome://gpu](chrome://gpu) 并提供图形硬件的报告文件
不清楚有没有包含用户敏感信息，如果有请自行剔除

**Comment by MianSoft** [comment]
[about-gpu-2024-01-31T08-11-05-246Z.txt](https://github.com/Steve-xmh/applemusic-like-lyrics/files/14108979/about-gpu-2024-01-31T08-11-05-246Z.txt)

**Comment by Linho1219** [comment]
此问题在旧版本上可以复现，在当前版本 AMLL Player 上无法复现。

我们认为这是旧版本遗留问题。现关闭此讨论。
